rasa_core/training/interactive.py

```diff
@@ -792,14 +805,60 @@ def _validate_action(action_name,  # type: Text
     ]
     answers = _ask_questions(questions, sender_id, endpoint)
     if not answers["action"]:
-        corrected_action = _request_action_from_user(predictions, sender_id,
-                                                     endpoint)
-        _correct_wrong_action(corrected_action, endpoint, sender_id,
+        action_name = _request_action_from_user(predictions, sender_id,
+                                                endpoint)
+
+    tracker = retrieve_tracker(endpoint, sender_id,
+                               EventVerbosity.AFTER_RESTART)
+
+    if (tracker.get('active_form', {}).get('name') and
+            action_name not in {tracker['active_form']['name'],
+                                ACTION_LISTEN_NAME}):
+        # notify the tracker that form was rejected
+        send_event(endpoint, sender_id,
+                   {"event": "action_execution_rejected",
+                    "name": tracker['active_form']['name']})
+    elif (action_name == tracker.get('active_form', {}).get('name') and
+            tracker.get('active_form', {}).get('rejected')):
+        # active form was chosen after it was rejected
+        # ask a user whether an input should be validated
+        q = ("Should '{}' validate user input to fill the slot '{}'?"
+             "".format(action_name, tracker.get("slots",
+                                                {}).get(REQUESTED_SLOT)))
+        validation_questions = [{
+            "name": "validation",
+            "type": "confirm",
+            "message": q
+        }]
+        form_answers = _ask_questions(validation_questions, sender_id,
+                                      endpoint)
+        if not form_answers["validation"]:
+            # notify form action to skip validation
+            send_event(endpoint, sender_id, {"event": "form_validation",
+                                             "validate": False})
+        elif not tracker.get('active_form', {}).get('validate'):
+            # handle contradiction with learned behaviour
+            q = ("WARNING: FormPolicy predicted no form validation "
```

Comment:
why is this a question? shouldn't this be a `logger.WARNING`? surely you don't want the user to answer anything there?

Reply:
this is the question because we use `print` function in interactive learning, and a I used a question here, so that it does not get scrolled and to make sure that user actually read it

Reply:
do we do this anywhere else in interactive learning? i feel like a logger warning is fine, no? it'll be confusing if the user has to press enter for something that doesn't require a confirmation

Reply:
hm, i'll see what it's like when i test out interactive learning